std/bigint.d

```diff
@@ -494,6 +495,14 @@ public:
                 return r;
             }
         }
+        else static if (is(Unqual!T == uint))
+        {
+            immutable uint u = absUnsign(y);
+            long rem = BigUint.modInt(data, u);
```

Comment:
Why can't you join this with the else clause below?
AFAICT the only thing that is different is the type of `rem`:

```d
static if (static if (is(Unqual!T == uint))
    alias T = long;
else
    alias T = int;
```